packages/@ama-sdk/core/src/fwk/mocks/base-mock-adapter.ts

```diff
@@ -33,8 +33,9 @@ export abstract class BaseMockAdapter implements MockAdapter {
   /**
    * @inheritDoc
    */
-  public getOperationId(request: EncodedApiRequest): string {
-    const object = getPath(request.basePath, this.pathObjects, request.method);
+  public async getOperationId(request: EncodedApiRequest): Promise<string> {
```

Comment:
This is actually a breaking change (on the return type) and can not be approved without deprecation of the previous function

Reply:
Have updated the code to add a new method to avoid a breaking change